components/spiffs/Makefile.projbuild

```diff
@@ -27,6 +33,7 @@ $(1)_bin: $(PARTITION_TABLE_BIN) $(SPIFFS_IMAGE_DEPENDS) | check_python_dependen
 	--page-size=$(CONFIG_SPIFFS_PAGE_SIZE) \
 	--obj-name-len=$(CONFIG_SPIFFS_OBJ_NAME_LEN) \
 	--meta-len=$(CONFIG_SPIFFS_META_LENGTH) \
+        $(FOLLOW_SYMLINKS) \
```

Comment:
Indentation seems to be wonky here, prefer aligning this with the subsequent lines.